### PR TITLE
Component/Footer Implementation

### PR DESCRIPTION
**Pull request by pedram-am**
The implementation includes the creation of the Footer component along with the necessary styling to ensure it seamlessly integrates into the application layout.

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+  return (
+    <div className="app">
+      <header className="app__header">
+        <h1 className="app__heading">CYF Hotel</h1>
+      </header>
+      <Bookings />
+      <Footer addresses={addresses} />
+    </div>
+  );
+};
```
Comment (HadikaMalik) [inline src/components/App/App.jsx:22]: Good work! Well done!

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <footer className="footer">
+    <ul className="footer__list">
+      {addresses.map((address, index) => (
+        <li key={index}>{address}</li>
+      ))}
+    </ul>
+  </footer>
+);
+
+export default Footer;
```
Comment (HadikaMalik) [inline src/components/Footer/Footer.jsx:14]: Looks perfect!

**Review by HadikaMalik — APPROVED** [review]
You code looks good and neat and is easy to understand. Well done!

**Comment by pedram-am** [comment]
@HadikaMalik Thank you for dedicating time to review the code. Your effort is greatly appreciated! 👍

File: src/components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.footer {
```
Comment (eliza-an) [inline src/components/Footer/Footer.scss:1]: do you have any instructions on the CSS for this page?
The page works but it looks a little choppy and slightly out of place form time to time. The footer is bulging out and quite thick taking up half of the page.

 Have a look at some real hotel pages for some inspiration on what would look good and cohesive. What do their footers look like? What makes them good?  Here are some examples: 
 https://www.oneandonlyresorts.com/cape-town
 https://www.samsaraubud.com/

**Review by eliza-an — COMMENTED** [review]
This looks good and functional. I understand the CSS is your last priority, but it will end up being important if this will be the project you show to your employers. Try to have a meeting as a team and see if you can look over the design if that's something you have time for

**Pull request merged**